.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: csharp
+sudo: false
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+    - zlib1g
+
+matrix:
+  include:
+    # Ubuntu 14.04
+    - os: linux 
+      dist: trusty
+      dotnet: "1.0.0-preview2-003131"
+      env: DOTNET_SDK_URL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-ubuntu-x64.latest.tar.gz
+    # OSX 10.11
+    - os: osx 
+      osx_image: xcode7.2
+      dotnet: "1.0.0-preview2-003131"
+      env: DOTNET_SDK_URL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-osx-x64.latest.tar.gz
+
+before install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update               ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install openssl      ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew link --force openssl ; fi
+
+install:
+  # Download .NET Core SDK and add to PATH
+  - export DOTNET_INSTALL_DIR="$PWD/.dotnetsdk"
+  - mkdir -p "$DOTNET_INSTALL_DIR"
+  - curl -L "$DOTNET_SDK_URL" | tar -xzv -C "$DOTNET_INSTALL_DIR"
+  - export PATH="$DOTNET_INSTALL_DIR:$PATH"
```

Comment:
yes, we can meanwhile work on `preview3` for fsproj.
we can later add `preview4` too when there are differences